ui/src/app/components/webcomponents/action-step-form.component.html

```diff
@@ -120,14 +120,14 @@
       <div class="p-0 field">
         <input
           [type]="inputType(argumentList[displayName.key])"
-          [placeholder]="argumentList[displayName.key]"
+          [placeholder]="argumentList[displayName.key].replace('java.lang.','')"
```

Comment:
can we move this to an utility function where the placeholder logic get implemented?
instead of hard coded text as **java.lang.**, refer from an extendable constant.
where, the other text which needs to be striped out from the placeholder can be added there.
@manikanta-reddy-t

Reply:
changed in latest commit @guganabu